### DISCUSSION TITLE
我的PCL似乎有点问题

### DISCUSSION THREAD

**Discussion by ZerkyLiu** [Meloong-Git/PCL#881] [提问] [answered]
如题，比如我的PCL明明是快照版，但更新却不需要输入更新密钥就能更新。还有不能拖动安装mod等......
这是不输入更新密钥更新的日志：
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/10384477/Log1.txt)
[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/10384478/Log2.txt)
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/10384479/Log3.txt)
[Log4.txt](https://github.com/Hex-Dragon/PCL2/files/10384480/Log4.txt)
[Log5.txt](https://github.com/Hex-Dragon/PCL2/files/10384481/Log5.txt)

**Comment by WForst-Breeze** [comment, marked as answer]
1. 在你输入过一次更新密钥后，若识别码不变，PCL2 不会再向你索要更新密钥（见 v2.1.1 更新日志）
2. 请详细描述问题（）不能拖拽安装 mod 的具体表现是什么呢？或许你可以看看 #749 是不是一样的问题

**Comment by ZerkyLiu** [comment]
@WForst-Breeze 我有一台电脑，每次更新都需要输入更新密钥，但另一台电脑上的PCL更新不用输入密钥

**Reply by WForst-Breeze** [reply]
这个我就不太清楚了（）因为按道理讲应该不会的

**Comment by ZerkyLiu** [comment]
@WForst-Breeze 至于不能拖动安装mod，我的情况倒是与 https://github.com/Hex-Dragon/PCL2/issues/749 的情况一样，我也是win11，拖动会显示一个红色的斜杠。但是我另一台win10系统的电脑就可以（两台电脑PCL均为快照2.4.5版本）

**Reply by WForst-Breeze** [reply]
那应该就是了，这个问题是系统原因，PCL2无法解决☹️

**Comment by ZS2147483647** [comment]
经过测试,win11专业版 22H2 Insider Preview 25272可以正常使用拖拽安装mod的功能[PCL快照2.4.5]

**Comment by ZerkyLiu** [comment]
我的系统是：
版本	Windows 11 专业工作站版
版本	22H2
安装日期	‎2022/‎5/‎19
操作系统版本	22623.1095
体验	Windows Feature Experience Pack 1000.22638.1000.0

**Comment by Pigeon0v0** [comment]
请打开系统 UAC 并 **不要** 使用管理员身份启动 PCL
